templates/components/form/form.jsx

```diff
@@ -67,12 +87,21 @@ var Form = React.createClass({
      var fields = Object.keys(this.props.fields).map(this.buildFormElement);
      return <div role="form">{fields}</div>;
   },
-  getClasses: function(field) {
+  getInputClasses: function(field) {
     return React.addons.classSet({
       'form-control': true,
       'has-error': !this.isValid(field),
+      'is-valid': this.isValid(field)
```

Comment:
why would we call this twice? `var valid = this.isValid(field); ... return { 'has-error': !valid, 'is-valid': valid }` but if these states are only ever each other's opposite, why would we even have an `is-valid` class rather than just styling based on whether `has-error` is present or not?